dipy/reconst/dti.py

```diff
@@ -1163,6 +1165,42 @@ def predict(self, gtab, S0=1):
         return tensor_prediction(self.model_params[..., 0:12], gtab, S0=S0)
 
 
+def iter_fit_tensor(step=1e4):
```

Comment:
I am going to go ahead and merge this PR because a lot of people are always using the master. Please add a Parameters section in another PR. And explain what step is. Is this the number of voxels for example?